manual/cops_vendor.md

```diff
@@ -1,5 +1,51 @@
 # Vendor
 
+## Vendor/RecursiveOpenStructGem
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | No | - | -
+
+This cop flags uses of the recursive-open-struct gem.
+
+RecursiveOpenStruct inherits from OpenStruct, which is now officially discouraged to be used
+for performance, version compatibility, and security issues.
+
+https://ruby-doc.org/stdlib-3.0.1/libdoc/ostruct/rdoc/OpenStruct.html#class-OpenStruct-label-Caveats
+
+## Vendor/RecursiveOpenStructUse
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | No | - | -
+
+This cop flags uses of RecursiveOpenStruct. RecursiveOpenStruct is a library used in the
+Wealthsimple ecosystem that is being phased out due to security issues.
+
+RecursiveOpenStruct inherits from OpenStruct, which is now officially discouraged to be used
+for performance, version compatibility, and security issues.
+
+### Examples
+
+```ruby
+# bad
+point = RecursiveOpenStruct.new(x: 0, y: 1)
+
+# good
+Point = Struct.new(:x, :y)
```

Comment:
did we want to encourage use of [`ResourceStruct`](https://github.com/alexriedler/resource-struct) instead? That seemed to work pretty well as a drop-in replacement for `RecursiveOpenStruct`

Reply:
ah actually I saw this is just from a previously added rule and you fixed the documentation generation